src/data-types-lib/index.js

```diff
@@ -100,6 +101,16 @@ export const
         return true;
       }
 
+      let customValidationResult = false;
+
+      if (validate) {
+        try {
+          customValidationResult = validate(value, instance)
```

Comment:
According to the documentation, custom **validate()** return value should be ignored.